### PR TITLE
Little test

### PR DESCRIPTION
**Pull request by DidoeS14**

### CODE REVIEW

File: SoftTechTest/SoftTechTest/SoftTechTest.cpp [hunk 1]
```diff
@@ -9,6 +9,8 @@ int main()
 	cout << "Suma=" << suma(a, n) << endl;
 	cout << "Max=" << max(a, n) << endl;
 	cout << "Average=" << average(a, n) << endl;
+	cout << "Test" << endl;
+
```
Comment (halilm00) [inline SoftTechTest/SoftTechTest/SoftTechTest.cpp:13]: Remove empty line

**Comment by Nunsamurai** [comment]
test

**Comment by halilm00** [comment]
> test

test comment

**Comment by VJecheva** [comment]
Браво, много добре!

**Pull request merged**